code/style.css

```diff
@@ -1,4 +1,207 @@
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
 }
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: rgb(232, 240, 242);
+  color: rgb(5, 55, 66)
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding-bottom: 0;
+  background-color: rgb(5, 55, 66, .90);
+  color: white;
+}
+
+h1 {
+  font-weight: normal;
+  font-size: 18px;
+  margin-left: 10px;
+}
+
+h2 {
+  text-align: center;
+  font-weight: normal;
+  font-size: 18px;
+}
+
+i {
+  font-size: x-large;
+}
+
+/* HEADER & NAVIGATION */
+.logo-container {
+  display: flex;
+  align-items: center;
+  margin-left: 10px;
+  padding-top: 5px;
+}
+
+.logo {
+  width: 60px;
+  }
+  
+.hidden-anchor {           /* the hidden anchor is needed to keep the margin between the fixed header and the first big-news-article*/
+  height: 90px;
+}
+
+
+.nav-link {                /* navbar is hidden on mobile */
+  display: none;
+}
+
+/* NEWS SECTION */
+.big-news-image, .other-news-image  {
+  display: block;
+  width: 87%;
+  max-width: 590px;
+  height: 73%;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 1px;
+  box-shadow: 3px 3px 4px grey;
+  background-color: white;
+}
+
+.big-news-text, .other-news-text {
+  width: 87%;
+  max-width: 590px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* CONTACT SECTION */
+.sm-icons {
+  width: 50%;
+  max-width: 634px;
+  margin: 50px auto 0 auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.contact-paragraph {
+  text-align: center;
+  font-size: 17px;
+  margin-top: 20px;
+}
+
+.fab {
+  color: rgb(5, 55, 66);
+}
+
+.fab:hover {
+  font-size: xx-large;                    /* hover effect: social media icons get bigger */
+}
+
+
+/* FOOTER */
+.footer-text {
+  margin-top: 50px;
+  font-size: 12px;
+  text-align: center;
+}
+
+
+
+
+
+
+/* ******MEDIA QUERIES****** */
+
+
+
+
+
+
+/* STYLE FOR TABLET ONLY */
+
+@media (min-width: 668px) and (max-width: 1024px) {
+  h1 {
+    display: none;                     /* sitename hidden on tablet */
+  }
+
+  #about {                             /* on tablet: show About and Blog on the left side of the navbar */
+    order: -1;
+  }
+
+  #blog {
+    order: -1;
+  }
+}
+
+
+/* STYLE FOR TABLET AND DESKTOP*/
+
+@media (min-width: 668px) {
+  header {                                
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 60px;
+  }
+  
+  .nav-link {                
+    display: block;
+    color:white;
+    text-decoration: none;
+  }
+
+  .nav-link:hover {                   /* the navigation-links get more visible on hovering */
+    font-weight: bold;
+    font-size: large;
+  }
+
+  .big-news-article {
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+  }
+
+  .big-news-text {
+    margin-left: 15px;
+  }
+
+  .other-news {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
It better write `grid-template-columns: 1fr; ` it is the same as you wrote just much shorter :)

Reply:
I I do that, than I have just one column instead of two.  
Did you mean this instead?:
`grid-template-columns: repeat(2, 1fr);`  :)

Reply:
Yeah exactly! You are right! I was thinking the grid is working as margin or padding, but it is quite different :)